inji-web-test/README.md

```diff
@@ -14,7 +14,14 @@ Ensure the following software is installed on the machine from where the automat
 1. Clone the repository by git clone https://github.com/mosip/inji-web.git
 2. Change directory by using command 'cd ../inji-web-test'  & Build the JAR file: `mvn clean package -DskipTests=true`
 3. The JAR file will be generated in the `target` directory.
-4. For running tests on Device Farm, use the JAR file with dependencies (`zip-with-dependencies`).
+
```

Comment:
can we explicitly mention, we required java 21 as a prerequisite

like this
https://github.com/mosip/mimoto/tree/develop?tab=readme-ov-file#build--run-for-developers